cmd/containerd-stress/main.go

```diff
@@ -216,7 +240,88 @@ func serve(c config) error {
 		}
 	}()
 	checkBinarySizes()
-	return test(c)
+	if c.CRI {
+		return criTest(c)
+	} else {
```

Comment:
Mind addressing the linter error so that the rest of CI can continue?

> indent-error-flow: if block ends with a return statement, so drop this else and outdent its block (revive)